query/src/servers/http/clickhouse_handler.rs

```diff
@@ -141,7 +145,17 @@ pub async fn clickhouse_handler_get(
         .await
         .map_err(InternalServerError)?;
 
+    let settings = session.get_settings();
+    for (k, v) in params.extra.iter() {
+        if settings.has_setting(k.as_str()) {
+            settings
+                .set_settings(k.to_string(), v.to_string(), false)
+                .map_err(BadRequest)?;
```

Comment:
to not affect usage of clickhouse clients, I suggest for now not return error when meeting an unkown setting, we can log first

for 2 reasons:

1. clickhouse has many other params not included in current StatementHandlerParams, [I will add them later]
2.  settings names of clickhouse and data bend are not the same.  [may need  to modify our setting or add a mapping/translation fn later]

Reply:
OK

Reply:
you can add comment and test for it

Reply:
I think about it. Unkown settings can't occur error. Only the setting value is wrong, the error occurs. I think it should throw error.

Reply:
We first judge by 'has_setting()' function.

Reply:
make sense